### PR TITLE
Creating a stub of the file for berlin

### LINKED ISSUE #25: Create a new file for 6axter82 hometown
## Add file for Berlin Germany
- [x] Create a branch
- [x] Add my file
- [x] Create a commit
- [x] Open a pull request
- [x] Have a discussion about the pull request
- [ ] Merge out pull request

### PR DESCRIPTION
**Pull request by 6axter82**
Which happens to be @6axter82's hometown.

This Pull Request closes #25

### CODE REVIEW

**Comment by lmalgeri** [comment]
Hello - Would love to visit Berlin!

**Comment by annadamson** [comment]
comment here?

File: places/berlin.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Berlin, Germany
+
+### Great Places to Eat
+
+### Fun things to do
```
Comment (annadamson) [inline places/berlin.md:5]: @6axter82 Do you have a specific recommendation for hiking in the Berlin area?

Comment (annadamson) [inline places/berlin.md:5]: This looks great, :shipit:

**Comment by 6axter82** [comment]
hey

**Pull request merged**